app/routes/application.js

```diff
@@ -172,7 +172,6 @@ export default Route.extend(
           // append mobile-wiki specific key
           surrogateKey = `${surrogateKey} ${surrogateKey}-mobile-wiki`;
           fastboot.get('response.headers').set('Surrogate-Key', surrogateKey);
-          fastboot.get('response.headers').set('X-Surrogate-Key', surrogateKey);
```

Comment:
@themech was there a good reason to add it in https://github.com/Wikia/mobile-wiki/pull/786/commits/74e2b583fc54dbb52e7f8357e75db91eede2882f?

Reply:
It was to make it compatible with other apps, that were using both `Surrogate-Key` and `X-Surrogate-Key` headers. The thing is that `Surrogate-Key` is not returned from Fastly with the response, so we thought it the `X-` version would help us with debugging/purging - you can just fetch any url and check the `X-Surrogate-Key` header to see how to purge a given wiki/app.
But luckily we can send the `Fastly-Debug` to Fastly so it returns the `Surrogate-Key`. This renders the `X-Surrogate-Key` header obsolete.